doc/testing.md

```diff
@@ -5,10 +5,60 @@ We strive to improve and maintain quality of the code with up-to-date samples, g
 
 ## Getting started
 
-Pilot tests require access to a Kubernetes cluster (version 1.5.2 or higher). Please
-configure your `kubectl` to point to a development cluster (e.g. minikube)
-before building or invoking the tests and add a symbolic link to your
-repository pointing to Kubernetes cluster credentials:
+### Create development cluster
+
+Pilot tests require access to a Kubernetes cluster (version 1.7.3 or higher). 
+
+Kubernetes [DynamicAdmissionControl](https://kubernetes.io/docs/admin/extensible-admission-controllers/#external-admission-webhooks) is required for transparent proxy injection (via initializer) and configuration validation. This is an alpha feature that must be explicitly enabled.
+
+[Role-Based Access Control (RBAC)](https://cloud.google.com/container-engine/docs/role-based-access-control) is optional. It must be explicitly enabled.
+
+* GKE
+
+```bash
+gcloud container clusters create NAME \               
+    --cluster-version=1.7.3 \
+    --enable-kubernetes-alpha \ 
+    --machine-type=n1-standard-2 \
+    --num-nodes=4 \
+    --zone=ZONE
+```
+
+Add `--no-enable-legacy-authorization` to the list of gcloud flags to fully enable RBAC in GKE. 
+
+Updates your kubeconfig file with appropriate credentials to point kubectl to the cluster created in GKE.
+
+```
+gcloud container clusters get-credentials NAME --zone=ZONE
+```
+
+Make sure you are using static client certificates before fetching cluster credentials:
+
+```
+gcloud config set container/use_client_certificate True
+```
+
+If RBAC was enabled, you must grant your user the ability to create authorization roles by running the following Kubernetes command:
+
+```bash
+kubectl create clusterrolebinding cluster-admin-binding --clusterrole=cluster-admin [--user=<user-name>]
+```
+
+* Minikube
+
+```bash
+minikube start \
+    --extra-config=apiserver.Admission.PluginNames="Initializers,NamespaceLifecycle,LimitRanger,ServiceAccount,DefaultStorageClass,ResourceQuota,GenericAdmissionWebhook" \
```

Comment:
If we included ResourceQuota, we should keep it at the end of the list to prevent incrementing quota usage prematurely. I will create a PR to fix this later.

Reply:
https://github.com/istio/pilot/pull/1221